src/hotspot/share/memory/metaspaceShared.hpp

```diff
@@ -239,20 +227,27 @@ class MetaspaceShared : AllStatic {
   static void init_misc_code_space();
   static address i2i_entry_code_buffers();
 
-  static void relocate_klass_ptr(oop o);
-  static Klass* get_relocated_klass(Klass *k, bool is_final=false);
-
   static void initialize_ptr_marker(CHeapBitMap* ptrmap);
 
   // This is the base address as specified by -XX:SharedBaseAddress during -Xshare:dump.
   // Both the base/top archives are written using this as their base address.
+  //
+  // During static dump: _requested_base_address == SharedBaseAddress.
+  //
+  // During dynamic dump: _requested_base_address is not always the same as SharedBaseAddress:
+  // - SharedBaseAddress is used for *reading the base archive*. I.e., CompactHashtable uses
+  //   it to convery offsets to pointers to Symbols in the base archive.
```

Comment:
Perhaps:
s/convery/convey/

Reply:
Thanks for spotting it. It's "convert". I'll fixed it :-)